### PR TITLE
[17.0][MIG] partner_survey: Migration to 17.0

### PR DESCRIPTION
**Pull request by chaule97**

### CODE REVIEW

File: partner_survey/views/res_partner_view.xml [hunk 1]
```diff
@@ -19,19 +19,19 @@
                     type="object"
                     icon="fa-list-alt"
                     name="action_view_surveys"
-                    attrs="{'invisible': ['|', '|', ('surveys_count', '=', 0), ('is_company', '=', True), ('surveys_invisible', '=', True)]}"
+                    invisible="surveys_count == 0 or is_company == True or surveys_invisible == True"
```
Comment (dzungtran89) [inline partner_survey/views/res_partner_view.xml]:
```suggestion
                    invisible="surveys_count == 0 or is_company or surveys_invisible"
```

File: partner_survey/views/res_partner_view.xml [hunk 2]
```diff
@@ -40,19 +40,19 @@
                     type="object"
                     icon="fa-list-alt"
                     name="action_view_surveys"
-                    attrs="{'invisible': ['|', '|', ('surveys_company_count', '=', 0), ('is_company', '=', False), ('surveys_company_invisible', '=', True)]}"
+                    invisible="surveys_company_count == 0 or is_company == False or surveys_company_invisible == True"
```
Comment (dzungtran89) [inline partner_survey/views/res_partner_view.xml]:
```suggestion
                    invisible="surveys_company_count == 0 or not is_company or surveys_company_invisible"
```

File: partner_survey/readme/CONTRIBUTORS.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
-- Damien Crier \<<damien.crier@camptocamp.com>\>
-- Akim Juillerat \<<akim.juillerat@camptocamp.com>\>
-- [Tecnativa](https://www.tecnativa.com):
-  - Rafael Blasco
-  - Antonio Espinosa
-  - Ernesto Tejeda
-  - Stefan Ungureanu
+* Javier Iniesta <javieria@antiun.com>
+* Damien Crier <damien.crier@camptocamp.com>
+* Akim Juillerat <akim.juillerat@camptocamp.com>
```
Comment (dzungtran89) [inline partner_survey/readme/CONTRIBUTORS.md:3]: no need to update the syntax of these lines, just append your own email

Comment (dzungtran89) [reply partner_survey/readme/CONTRIBUTORS.md:3]: some minor changes:
- follow to this [guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#technical-method-to-migrate-a-module-from-160-to-170-branch) to update the the precommit message and PR title

Comment (chaule97) [reply partner_survey/readme/CONTRIBUTORS.md:3]: > no need to update the syntax of these lines, just append your own email

I just append my own email but pre-commit fix those syntax

Comment (dzungtran89) [reply partner_survey/readme/CONTRIBUTORS.md:3]:
```- Javier Iniesta \<<javieria@antiun.com>\>
- Damien Crier \<<damien.crier@camptocamp.com>\>
- Akim Juillerat \<<akim.juillerat@camptocamp.com>\>
- [Tecnativa](https://www.tecnativa.com):
  - Rafael Blasco
  - Antonio Espinosa
  - Ernesto Tejeda
  - Stefan Ungureanu
- Chau Le \<<chaulb@trobz.com>\>
```

Comment (dzungtran89) [reply partner_survey/readme/CONTRIBUTORS.md:3]: also, please fix the pre-commit message according to this [guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#technical-method-to-migrate-a-module-from-160-to-170-branch)

**Pull request closed without merging**